home-page.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://shed-ui.netlify.app/component.css');
+@import url(/utils.css);
+
+.nav-icon-container{
+    margin-left: auto;
+}
+.top{
+    margin-top: 10rem;
+}
+.hero-container{
+    padding-inline: 5rem;
+    gap: 2rem;
+}
+.quiz-heading{
+    flex-direction: column;
```

Comment:
Suggestion : Since flex gets used in a lot of places in-general, you can try creating a utility css file for flex with classes containing all the styles and use it from there.